README.md

```diff
@@ -40,6 +40,7 @@ Here's a list of available language APIs for obs-websocket:
 - Rust: [obws](https://github.com/dnaka91/obws/tree/v5-api) by dnaka91
 - Godot 3.4.x: [obs-websocket-gd](https://github.com/you-win/obs-websocket-gd) by you-win
 - Javascript (Node and web): [obs-websocket-js](https://github.com/obs-websocket-community-projects/obs-websocket-js) by OBS Websocket Community
+- Dart/Flutter (all platforms - cli, desktop, mobile and web): [obs_websocket](https://github.com/faithoflifedev/easy_obs_websocket) by faithoflifedev
```

Comment:
Maybe add the name of you're library instead of `obs_websocket`

Reply:
The repository is called `easy_obs_websocket` but it is included as `obs_websocket` in the dependencies configuration.